dashboard/management/commands/site.py

```diff
@@ -0,0 +1,25 @@
+from django.core.management.base import BaseCommand
+from django.contrib.sites.models import Site
+from django.conf import settings
+
+
+class Command(BaseCommand):
+    def add_arguments(self, parser):
+        parser.add_argument('--domain', dest='domain', type=str, required=True)
+        parser.add_argument('--name', dest='name', type=str, required=True)
+        parser.add_argument('--new', dest='new', type=bool, required=False)
+
+    def handle(self, *args, **options):
+        domain = options.get('domain')
+        name = options.get('name')
+        new = options.get('new')
```

Comment:
when will the "new" option be used? Did not see an example in start.sh. May need documentation here.

Reply:
It isn't being used, but it seemed like a site command (that could be run from the container shell) should be a bit more flexible than just updating the default site. Do you want me to add a comment?

Reply:
See latest commit, and let me know if that addresses your concern.

Reply:
Perfect!